CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
+- Support `:not(…)` as an entire selector
+  ([#469](https://github.com/MyIntervals/emogrifier/pull/469),
```

Comment:
#469 is an issue, not a PR (although GitHub has redirects in place for this).

Reply:
Good catch.  A result of copying and pasting and changing just the numbers.